commands/blacklist/addbl.js

```diff
@@ -0,0 +1,17 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+const { blacklist } = require('../../utility/blacklist');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('addbl')
+        .setDescription('Add a message to the blacklist')
+        .addStringOption(option =>
+            option.setName('message_id')
+            .setDescription('The ID of a message you want to remove from the blacklist')),
+    async execute(interaction) {
```

Comment:
Can we add error handling to this? What happens if we give it an invalid `message_id`?